<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from dual_arm.xacro                 | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="dual_arm">
  <!--Constant define-->
  <!--base define-->
  
  <link name="dummy">
    </link>
  


  <joint name="dummy_center_joint" type="fixed">
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <parent link="dummy"/>
    <child link="cloud_platform"/>
  </joint>
  <link name="cloud_platform">
    </link>

  <joint name="cloud_platform_joint" type="continuous">
    <origin rpy="0 0 -1.570795" xyz="0 0 0"/>
    <parent link="cloud_platform"/>
    <child link="base_link"/>
    <axis xyz="0 0 1"/>
    <!--limit effort="60" lower="-3.14159" upper="4.14159" velocity="3.14"/-->
  </joint>

  <link name="base_link">
    <inertial>
      <origin rpy="0 0 0" xyz="1.3843E-15 1.0159E-16 0.052813"/>
      <mass value="1.088"/>
      <inertia ixx="0.0062111" ixy="-2.1701E-18" ixz="-3.8041E-17" iyy="0.019573" iyz="2.4964E-18" izz="0.018603"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/center_link.stl"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/center_link.stl"/>
      </geometry>
    </collision>
  </link>
  <link name="kinect2_link">
    </link>
  <joint name="camera_center_joint" type="fixed">
    <origin rpy="3.132523668443418 -0.08587519624949608 1.574670554067391" xyz="1.726742473702382041e-02 -4.821492818581604722e-01 7.785277826224238362e-01"/>
    <parent link="dummy"/>
    <child link="kinect2_link"/>
  </joint>
  <joint name="left_base_joint" type="fixed">
    <origin rpy="-1.570795 0 1.570795" xyz="-0.075 0 0.095"/>
    <parent link="base_link"/>
    <child link="left_base_link"/>
  </joint>
  <link name="left_base_link">
    <inertial>
      <origin rpy="0 0 0" xyz="-7.07923322916521E-05 -3.15078560279351E-07 0.00211490550882476"/>
      <mass value="0.947876033180305"/>
      <inertia ixx="0.0017318413107461" ixy="3.14952850900379E-10" ixz="2.06131736991631E-06" iyy="0.00151929091543902" iyz="-2.03157503363577E-07" izz="0.00288656909579778"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/base_link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/base_link.STL"/>
      </geometry>
    </collision>
  </link>
  <link name="left_1_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="0.0136179271893209 7.04702633335797E-07 -0.00891671768932772"/>
      <mass value="0.427787775289708"/>
      <inertia ixx="0.000328336335162208" ixy="1.39942675254359E-08" ixz="-4.96624559527467E-05" iyy="0.000349499347099752" iyz="7.16862445800443E-09" izz="0.000398779291439988"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/1_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/1_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="left_axis_joint_1" type="revolute">
    <origin rpy="0 0 1.5708" xyz="0 -1.7907E-05 0.1205"/>
    <parent link="left_base_link"/>
    <child link="left_1_Link"/>
    <axis xyz="0 0 -1"/>
    <limit effort="60" lower="-2.4" upper="2.4" velocity="3.14"/>
  </joint>
  <link name="left_2_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="0.0282477009101944 4.3906549022065E-07 -0.00996354944497273"/>
      <mass value="0.474122227914849"/>
      <inertia ixx="0.000348140976643854" ixy="6.62975404584032E-09" ixz="-3.65184694791613E-05" iyy="0.000363227053478232" iyz="-1.49821649022608E-08" izz="0.00041732509280613"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/2_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/2_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="left_axis_joint_2" type="revolute">
    <origin rpy="-1.5708 3.1416 -1.5708" xyz="0.1165 0 0"/>
    <parent link="left_1_Link"/>
    <child link="left_2_Link"/>
    <axis xyz="0 0 -1"/>
    <limit effort="60" lower="-2.4" upper="0.1" velocity="3.14"/>
  </joint>
  <link name="left_3_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="0.0550000657149 1.02706571201196E-05 -0.0873823729588203"/>
      <mass value="0.150735858555544"/>
      <inertia ixx="6.48442797417871E-05" ixy="-7.37989542660507E-09" ixz="-1.63575029334465E-07" iyy="7.51775686938988E-05" iyz="-6.30391436593442E-09" izz="8.14430313397472E-05"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/3_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/3_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="left_axis_joint_3" type="revolute">
    <origin rpy="-0.00039303 0 0" xyz="0.20352 2.2698E-05 0.03825"/>
    <parent link="left_2_Link"/>
    <child link="left_3_Link"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-2.4" upper="2.4" velocity="3.14"/>
  </joint>
  <link name="left_4_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="0.00316829014378384 -2.51807207740376E-05 -0.065762603358388"/>
      <mass value="0.0868240169253312"/>
      <inertia ixx="3.69273389374954E-05" ixy="-7.74427846366249E-09" ixz="6.58227930408113E-06" iyy="3.88095070665613E-05" iyz="-3.86379121232784E-09" izz="4.25312703267998E-05"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/4_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/4_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="left_axis_joint_4" type="revolute">
    <origin rpy="-3.1412 0 0" xyz="0.173 1.58E-05 -0.1557"/>
    <parent link="left_3_Link"/>
    <child link="left_4_Link"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-2.4" upper="2.4" velocity="3.14"/>
  </joint>
  <link name="left_5_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="0.00301334222958494 2.51282272504116E-05 -0.0271578777359305"/>
      <mass value="0.0861219565127689"/>
      <inertia ixx="3.66130359688528E-05" ixy="-7.69517877575345E-09" ixz="6.39794733105846E-06" iyy="3.815099630915E-05" iyz="3.73716337127711E-09" izz="4.20198733154724E-05"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/5_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/5_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="left_axis_joint_5" type="revolute">
    <origin rpy="3.1416 1.5708 0" xyz="0 0 -0.11743"/>
    <parent link="left_4_Link"/>
    <child link="left_5_Link"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-2.4" upper="2.4" velocity="3.14"/>
  </joint>
  <link name="left_6_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="2.43062335152366E-05 1.80532182430252E-05 0.0293761927296333"/>
      <mass value="0.0341435122227797"/>
      <inertia ixx="9.69578526103206E-06" ixy="-7.64383826654747E-09" ixz="-3.2515655925902E-09" iyy="1.02214035330215E-05" iyz="6.36682596346152E-09" izz="1.67442220589365E-05"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/6_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/6_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="left_axis_joint_6" type="revolute">
    <origin rpy="3.1412 1.5708 0" xyz="0.019524 2.2696E-05 -0.079176"/>
    <parent link="left_5_Link"/>
    <child link="left_6_Link"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-2.4" upper="2.4" velocity="3.14"/>
  </joint>
  <!--gazebo reference="left_base_link">
    <material>Gazebo/White</material>
  </gazebo>
  <gazebo reference="left_1_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <gazebo reference="left_2_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <gazebo reference="left_3_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <gazebo reference="left_4_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <gazebo reference="left_5_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <gazebo reference="left_6_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <transmission name="left_axis_joint_1_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="left_axis_joint_1">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="left_axis_joint_1_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <transmission name="left_axis_joint_2_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="left_axis_joint_2">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="left_axis_joint_2_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <transmission name="left_axis_joint_3_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="left_axis_joint_3">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="left_axis_joint_3_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <transmission name="left_axis_joint_4_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="left_axis_joint_4">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="left_axis_joint_4_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <transmission name="left_axis_joint_5_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="left_axis_joint_5">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="left_axis_joint_5_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <transmission name="left_axis_joint_6_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="left_axis_joint_6">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="left_axis_joint_6_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission-->
  <joint name="right_base_joint" type="fixed">
    <origin rpy="1.570795 0 1.570795" xyz="0.075 0 0.095"/>
    <parent link="base_link"/>
    <child link="right_base_link"/>
  </joint>
  <link name="right_base_link">
    <inertial>
      <origin rpy="0 0 0" xyz="-7.07923322916521E-05 -3.15078560279351E-07 0.00211490550882476"/>
      <mass value="0.947876033180305"/>
      <inertia ixx="0.0017318413107461" ixy="3.14952850900379E-10" ixz="2.06131736991631E-06" iyy="0.00151929091543902" iyz="-2.03157503363577E-07" izz="0.00288656909579778"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/base_link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/base_link.STL"/>
      </geometry>
    </collision>
  </link>
  <link name="right_1_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="0.0136179271893209 7.04702633335797E-07 -0.00891671768932772"/>
      <mass value="0.427787775289708"/>
      <inertia ixx="0.000328336335162208" ixy="1.39942675254359E-08" ixz="-4.96624559527467E-05" iyy="0.000349499347099752" iyz="7.16862445800443E-09" izz="0.000398779291439988"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/1_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/1_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="right_axis_joint_1" type="revolute">
    <origin rpy="0 0 -1.5708" xyz="0 -1.7907E-05 0.1205"/>
    <parent link="right_base_link"/>
    <child link="right_1_Link"/>
    <axis xyz="0 0 -1"/>
    <limit effort="60" lower="-2.4" upper="2.4" velocity="3.14"/>
  </joint>
  <link name="right_2_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="0.0282477009101944 4.3906549022065E-07 -0.00996354944497273"/>
      <mass value="0.474122227914849"/>
      <inertia ixx="0.000348140976643854" ixy="6.62975404584032E-09" ixz="-3.65184694791613E-05" iyy="0.000363227053478232" iyz="-1.49821649022608E-08" izz="0.00041732509280613"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/2_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/2_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="right_axis_joint_2" type="revolute">
    <origin rpy="-1.5708 0 -1.5708" xyz="0.1165 0 0"/>
    <parent link="right_1_Link"/>
    <child link="right_2_Link"/>
    <axis xyz="0 0 -1"/>
    <limit effort="60" lower="-0.1" upper="1.54" velocity="3.14"/>
  </joint>
  <link name="right_3_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="0.0550000657149 1.02706571201196E-05 -0.0873823729588203"/>
      <mass value="0.150735858555544"/>
      <inertia ixx="6.48442797417871E-05" ixy="-7.37989542660507E-09" ixz="-1.63575029334465E-07" iyy="7.51775686938988E-05" iyz="-6.30391436593442E-09" izz="8.14430313397472E-05"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/3_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/3_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="right_axis_joint_3" type="revolute">
    <origin rpy="-0.00039303 0 0" xyz="0.20352 2.2698E-05 0.03825"/>
    <parent link="right_2_Link"/>
    <child link="right_3_Link"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-2.4" upper="2.4" velocity="3.14"/>
  </joint>
  <link name="right_4_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="0.00316829014378384 -2.51807207740376E-05 -0.065762603358388"/>
      <mass value="0.0868240169253312"/>
      <inertia ixx="3.69273389374954E-05" ixy="-7.74427846366249E-09" ixz="6.58227930408113E-06" iyy="3.88095070665613E-05" iyz="-3.86379121232784E-09" izz="4.25312703267998E-05"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/4_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/4_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="right_axis_joint_4" type="revolute">
    <origin rpy="-3.1412 0 0" xyz="0.173 1.58E-05 -0.1557"/>
    <parent link="right_3_Link"/>
    <child link="right_4_Link"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-2.4" upper="2.4" velocity="3.14"/>
  </joint>
  <link name="right_5_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="0.00301334222958494 2.51282272504116E-05 -0.0271578777359305"/>
      <mass value="0.0861219565127689"/>
      <inertia ixx="3.66130359688528E-05" ixy="-7.69517877575345E-09" ixz="6.39794733105846E-06" iyy="3.815099630915E-05" iyz="3.73716337127711E-09" izz="4.20198733154724E-05"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/5_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/5_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="right_axis_joint_5" type="revolute">
    <origin rpy="3.1416 1.5708 0" xyz="0 0 -0.11743"/>
    <parent link="right_4_Link"/>
    <child link="right_5_Link"/>
    <axis xyz="0 0 1"/>
    <limit effort="30" lower="-2.4" upper="2.4" velocity="3.14"/>
  </joint>
  <link name="right_6_Link">
    <inertial>
      <origin rpy="0 0 0" xyz="2.43062335152366E-05 1.80532182430252E-05 0.0293761927296333"/>
      <mass value="0.0341435122227797"/>
      <inertia ixx="9.69578526103206E-06" ixy="-7.64383826654747E-09" ixz="-3.2515655925902E-09" iyy="1.02214035330215E-05" iyz="6.36682596346152E-09" izz="1.67442220589365E-05"/>
    </inertial>
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/6_Link.STL"/>
      </geometry>
      <material name="">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://gluon/meshes/6_Link.STL"/>
      </geometry>
    </collision>
  </link>
  <joint name="right_axis_joint_6" type="revolute">
    <origin rpy="3.1412 1.5708 0" xyz="0.019524 2.2696E-05 -0.079176"/>
    <parent link="right_5_Link"/>
    <child link="right_6_Link"/>
    <axis xyz="0 0 -1"/>
    <limit effort="30" lower="-2.4" upper="2.4" velocity="3.14"/>
  </joint>
  <!--gazebo reference="right_base_link">
    <material>Gazebo/White</material>
  </gazebo>
  <gazebo reference="right_1_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <gazebo reference="right_2_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <gazebo reference="right_3_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <gazebo reference="right_4_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <gazebo reference="right_5_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <gazebo reference="right_6_Link">
    <material>Gazebo/White</material>
    <selfCollide>true</selfCollide>
  </gazebo>
  <transmission name="right_axis_joint_1_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="right_axis_joint_1">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="right_axis_joint_1_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <transmission name="right_axis_joint_2_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="right_axis_joint_2">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="right_axis_joint_2_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <transmission name="right_axis_joint_3_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="right_axis_joint_3">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="right_axis_joint_3_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <transmission name="right_axis_joint_4_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="right_axis_joint_4">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="right_axis_joint_4_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <transmission name="right_axis_joint_5_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="right_axis_joint_5">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="right_axis_joint_5_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <transmission name="right_axis_joint_6_trans">
    <type>transmission_interface/SimpleTransmission</type>
    <joint name="right_axis_joint_6">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
    </joint>
    <actuator name="right_axis_joint_6_motor">
      <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
      <mechanicalReduction>1</mechanicalReduction>
    </actuator>
  </transmission>
  <gazebo>
    <plugin filename="libgazebo_ros_control.so" name="gazebo_ros_control"/>
  </gazebo-->
</robot>

